test/regex/emptyregex.chpl

```diff
@@ -16,12 +16,14 @@ for (i, (match,)) in zip(matches.domain, matches) {
 
 writeln();
 
-for x in compile("a").split("babbbba") do
+var re = new regex("a");
+for x in re.split("babbbba") do
```

Comment:
Yeah, that is sort of the only question that came up while changing tests. To be sure, you know that it'll require parens around the initializer, right? ie.

```chapel
for x in (new regex("a")).split("babbbba") do
```

I have a slight dislike towards this because of those extra parens. My eyes parse a leading `(` as if the whole expression until the next space will be parenthesized. But that's not the case.

Reply:
Yes  about parens. Given that we have both styles exercised in our tests, it is really immaterial which way we go on this particular test.